pilot/lib/pilot/options.rb

```diff
@@ -156,7 +156,11 @@ def self.available_options
                                      type: Array,
                                      verify_block: proc do |value|
                                        UI.user_error!("Could not evaluate array from '#{value}'") unless value.kind_of?(Array)
-                                     end)
+                                     end),
+        FastlaneCore::ConfigItem.new(key: :group_name,
+                                     env_name: "PILOT_GROUP_NAME",
+                                     description: "The name of the group being made for external testers",
```

Comment:
Could we maybe make a more specific name and description for this option? Just dont want people to confuse it with the `:groups` option above.